### PR TITLE
feat(styles): updated sizing utility (no tokens)

### PR DESCRIPTION
**Pull request by myrta2302**

### CODE REVIEW

File: packages/documentation/src/stories/utilities/sizing/sizing.stories.ts [hunk 1]
```diff
@@ -66,7 +78,10 @@ type Story = StoryObj;
 function renderSizing(args: Args) {
   const maximumHeight = args.maxHeight && args.maxHeight !== 'null' ? `mh-${args.maxHeight}` : '';
```
Comment (alizedebray) [inline packages/documentation/src/stories/utilities/sizing/sizing.stories.ts:79]: This is named "none" in the options (same for all classes bellow) 
```suggestion
  const maximumHeight = args.maxHeight && args.maxHeight !== 'none' ? `mh-${args.maxHeight}` : '';
```

File: packages/styles/src/utilities/_variables.scss [hunk 2]
```diff
@@ -22,6 +22,144 @@
...
+  'max-width': (
+    property: max-width,
+    class: mw,
+    values: (
+      0: 0,
+      25: 25%,
+      50: 50%,
+      75: 75%,
+      100: 100%,
+      auto: auto,
```
Comment (alizedebray) [inline packages/styles/src/utilities/_variables.scss:58]:
```suggestion
```

`max-width: auto` is invalid CSS.

**Review by alizedebray — CHANGES_REQUESTED** [review]

File: packages/documentation/src/stories/utilities/sizing/sizing.docs.mdx
Comment (alizedebray) [inline packages/documentation/src/stories/utilities/sizing/sizing.docs.mdx:1]: Can you check the headings levels? 

Also, there is a section about responsive classes but the classes are not responsive, can you either remove the section or make the classes responsive.

**Review by alizedebray — CHANGES_REQUESTED** [review]

File: packages/documentation/src/stories/utilities/sizing/sizing.docs.mdx
Comment (myrta2302) [reply packages/documentation/src/stories/utilities/sizing/sizing.docs.mdx:1]: ok added

File: packages/documentation/src/stories/utilities/spacing/spacing.stories.ts
[the remark this answers is not in the document]
Comment (myrta2302) [reply packages/documentation/src/stories/utilities/spacing/spacing.stories.ts:1]: I am not sure how. I tried to go to history and revert any commit that had the spacing file in it to remove it, but i ve probably missed something.

[the remark this answers is not in the document]
Comment (myrta2302) [reply packages/documentation/src/stories/utilities/spacing/spacing.stories.ts:1]: ok found it :)

**Comment by myrta2302** [comment]
Still getting the error regarding logo.cy

**Comment by oliverschuerch** [comment]
naming issues

**Pull request closed without merging**